### PR TITLE
Module 0.2

### PR DESCRIPTION
**Pull request by Sumit2280**

### CODE REVIEW

File: todo-app/src/Components/ToDoList.tsx [hunk 1]
```diff
@@ -1,90 +1,101 @@
...
 import IToDo from "../interfaces/Todo";
-
+import useFetch from "../hooks/useFetch";
 
 const ToDoList = () => {
-  const [todo, setToDo] = useState('');
-  const [todolist, setToDoList] = useState<IToDo[]>([])
+  const [todo, setToDo] = useState("");
+  const [todolist, setToDoList] = useState<IToDo[]>([]);
+  const url = "http://localhost:8000/todo";
```
Comment (MyLanguagePlug) [inline todo-app/src/Components/ToDoList.tsx:9]: Host URL : Try to save it in env

Comment (MyLanguagePlug) [reply todo-app/src/Components/ToDoList.tsx:9]: And try to save in header of axios

File: todo-app/src/hooks/useFetch.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import IToDo from "../interfaces/Todo";
+
+const useFetch = (url: string) => {
```
Comment (MyLanguagePlug) [inline todo-app/src/hooks/useFetch.tsx:5]: This hook will only work for get apis what about post,  put and delete ?

Comment (Sumit2280) [reply todo-app/src/hooks/useFetch.tsx:5]: working on it in the third module

**Pull request merged**